docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | `Warehouse`, `Stage`, `Promotion` | Arbitrary string (acts as a token) | Triggers reconciliation of the resource when the value changes. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the current `Freight`. |
+| `kargo.akuity.io/abort` | `Stage` | Verification ID | Aborts an in-progress `Freight` verification. |
+| `kargo.akuity.io/description` | Any | Free-form string | Human-readable description displayed in the UI. |
+| `kargo.akuity.io/color` | `Stage` | Hex color code (e.g. `#ff8800`) | Optional cosmetic color used in UI. |
+| `kargo.akuity.io/authorized-stage` | `Argo CD Application` | `<project>:<stage>` | Indicates which `Stage` is authorized to manage the `Application` resource. |
+
+:::warning
+Avoid setting system-managed annotations manually unless you're explicitly debugging internal behavior.
+:::
+
+### RBAC Annotations
```

Comment:
Since there are only two entries in the table below, I feel like "RBAC Annotations" is perhaps not worthy of its own distinct section. Let's just move these up into the previous section's table.